src/vehicle_display.cpp

```diff
@@ -379,4 +391,37 @@ void vehicle::print_fuel_indicator( const catacurses::window &win, int y, int x,
     if( desc ) {
         wprintz( win, c_light_gray, " - %s", item::nname( fuel_type ) );
     }
+    if( verbose ) {
+        auto fuel_data = fuel_usages.find( fuel_type );
+        int rate = 0;
+        std::string units;
+        if( fuel_data != fuel_usages.end() ) {
+            rate = consumption_per_hour( fuel_type, fuel_data->second );
+            units = _( "mL" );
```

Comment:
Do we really need milliliters here? The tanks capacity in the vehicle examination screen is in liters.
Furthermore, players may use cups or quarters for volumes.

Reply:
I'm just going to put it in percents for the general display, and leave mL for the debug display.